db/migrate/20240626175659_create_votes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateVotes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :votes do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :votable, polymorphic: true, null: false
```

Comment:
Like I said in the model, these two fields should be unique together so I would also add a unique index to validate this on the database side.